File: mobile/app/src/main/java/dam/isi/frsf/utn/edu/ar/delivery/model/Review.java
package dam.isi.frsf.utn.edu.ar.delivery.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Review implements Serializable
{

    @SerializedName("rating")
    @Expose
    private Integer rating;
    @SerializedName("comment")
    @Expose
    private String comment = "";
    @SerializedName("img")
    @Expose
    private String img = "";
    private final static long serialVersionUID = 2881016317022269792L;

    public Integer getRating() {
        return rating;
    }

    public void setRating(Integer rating) {
        this.rating = rating;
    }

    public Review withRating(Integer rating) {
        this.rating = rating;
        return this;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public Review withComment(String comment) {
        this.comment = comment;
        return this;
    }

    public String getImg() {
        return img;
    }

    public void setImg(String img) {
        this.img = img;
    }

    public Review withImg(String img) {
        this.img = img;
        return this;
    }

}

File: mobile/app/src/main/java/dam/isi/frsf/utn/edu/ar/delivery/model/Review.kt
package dam.isi.frsf.utn.edu.ar.delivery.model

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Review(
        @SerializedName("rating")
        @Expose
        var rating: Int = 0,
        @SerializedName("comment")
        @Expose
        var comment: String = "",
        @SerializedName("img")
        @Expose
        var img: String = ""
) : Parcelable {
    fun withRating(rating: Int): Review {
        this.rating = rating
        return this
    }

    fun withComment(comment: String): Review {
        this.comment = comment
        return this
    }

    fun withImg(img: String): Review {
        this.img = img
        return this
    }

    companion object {
        @JvmField val CREATOR: Parcelable.Creator<Review> = object : Parcelable.Creator<Review> {
            override fun createFromParcel(source: Parcel): Review = Review(source)
            override fun newArray(size: Int): Array<Review?> = arrayOfNulls(size)
        }
    }

    constructor(source: Parcel) : this(
    source.readInt(),
    source.readString(),
    source.readString()
    )

    override fun describeContents() = 0

    override fun writeToParcel(dest: Parcel, flags: Int) {
        dest.writeInt(rating)
        dest.writeString(comment)
        dest.writeString(img)
    }
}